streams/src/test/java/org/apache/kafka/streams/integration/ResetPartitionTimeIntegrationTest.java

```diff
@@ -156,7 +157,7 @@ public void shouldPreservePartitionTimeOnKafkaStreamRestart() {
             assertThat(lastRecordedTimestamp, is(5000L));
         } finally {
             kafkaStreams.close();
-            cleanStateAfterTest(CLUSTER, kafkaStreams);
+            quietlyCleanStateAfterTest(CLUSTER, kafkaStreams);
```

Comment:
nit: we can put kafkaStreams in a try block.

Reply:
Unfortunately, we generally can't use try-with-resources for these tests, since that makes the `kafkaStreams` reference out of scope for the finally block. We'd have to allow a reference to kafkaStreams to escape the try {} block to reference it either in finally or in an After method, which is just as messy as it currently is, if not more.